Uplift/Views/GymDetail/Facilities/EquipmentListItemCell.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  EquipmentListItemCell.swift
+//  Uplift
+//
+//  Created by Yana Sang on 10/9/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import SnapKit
+import UIKit
+
+class EquipmentListItemCell: ListItemCollectionViewCell<EquipmentCategory> {
+
+    // MARK: - Public static vars
+    static let identifier = Identifiers.gymEquipmentCell
+
+    // MARK: - Private view vars
+    private let titleLabel = UILabel()
+    private let equipmentList = UITextView()
+    private let quantityList = UITextView()
+
+    // MARK: - Private layout vars
+    private let listsSpacing: CGFloat = 12
+    private var equipmentListTrailingConstraint: Constraint?
+
+    // MARK: - Private data vars
+    private var equipment: [Equipment] = []
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        // BORDER
+        contentView.layer.cornerRadius = 4
+        contentView.layer.backgroundColor = UIColor.white.cgColor
+
+        // SHADOWING
+        contentView.layer.shadowColor = UIColor(red: 0, green: 0, blue: 0, alpha: 0.16).cgColor
+        contentView.layer.shadowOffset = CGSize(width: 0.0, height: 0)
+        contentView.layer.shadowRadius = 3.0
+        contentView.layer.shadowOpacity = 1.0
+        contentView.layer.masksToBounds = false
+
+        setupViews()
+        setupConstraints()
+    }
+
+    // MARK: - Public configure
+    override func configure(for equipmentCategory: EquipmentCategory) {
+        super.configure(for: equipmentCategory)
+
+        titleLabel.text = equipmentCategory.categoryName
+        equipment = equipmentCategory.equipment
+        equipmentList.attributedText = generateListString(for: equipment.map { $0.name })
+        quantityList.attributedText = generateListString(for: equipment.map { $0.quantity }, alignment: .right)
+        quantityList.sizeToFit()
+
+        equipmentListTrailingConstraint?.update(offset: -listsSpacing)
```

Comment:
why do we need this `trailingConstraint`? it doesn't seem like `listsSpacing` ever gets changed